src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -725,14 +565,31 @@ public partial class SqlDataReader : System.Data.Common.DbDataReader, System.IDi
         public override object GetValue(int i) { throw null; }
         public override int GetValues(object[] values) { throw null; }
         public virtual System.Xml.XmlReader GetXmlReader(int i) { throw null; }
-        public override bool IsDBNull(int i) { throw null; }
         protected internal bool IsCommandBehavior(System.Data.CommandBehavior condition) { throw null; }
+        public override bool IsDBNull(int i) { throw null; }
         public override System.Threading.Tasks.Task<bool> IsDBNullAsync(int i, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override bool NextResult() { throw null; }
         public override System.Threading.Tasks.Task<bool> NextResultAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
         public override bool Read() { throw null; }
         public override System.Threading.Tasks.Task<bool> ReadAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int i) { throw null; }
```

Comment:
Why would we add this here? Isn't it redundant since the class implements IDataReader?

Reply:
The method is needed if you want to hang an attribute on it and since it's one that directly affects the visibility of this method to intellisense it seemed like a good idea.